Tests/pytest/App/PyImports/test_ProjectSentinel.py

```diff
@@ -175,6 +175,18 @@ def test_create_project_zip():
         isSaved, saveName2 = create_project_zip('Dummy/Dir', FILE)
     temp1.cleanup()
 
+def test_LoadExperiment_cif():
+    model = ProjectControl()
+    model.loadExperiment("test_path", "boom (*.cif) boom")
```

Comment:
It is not needed, but makes the test more difficult for the function to pass. The current implementation checks for (*.cif), so that the remaining name filter can be changed without affecting the functionality.